chia/full_node/mempool_manager.py

```diff
@@ -424,9 +429,11 @@ async def validate_spend_bundle(
                 is_eligible_for_dedup=is_eligible_for_dedup,
                 spend_additions=spend_additions,
                 is_eligible_for_ff=is_eligible_for_ff,
+                ff_puzzle_hash=bytes32(spend.puzzle_hash),
```

Comment:
do you set the `ff_puzzle_hash` unconditionally here? I would have thought you only set it for ff-spends.